src/Distribution/Server/Features/UserNotify.hs

```diff
@@ -899,36 +898,103 @@ userNotifyFeature ServerEnv{serverBaseURI, serverCron}
                 ]
               <> EmailContentList (map renderPkgLink revDeps)
 
-    sendNotifyEmailAndDelay :: Users.Users -> (UserId, (T.Text, EmailContent)) -> IO ()
-    sendNotifyEmailAndDelay users (uid, (subject, emailContent)) = do
-        mudetails <- queryUserDetails uid
-        case mudetails of
-             Nothing -> return ()
-             Just (AccountDetails{accountContactEmail=eml, accountName=aname})-> do
-                 let mailFrom = Address (Just (T.pack "Hackage website"))
-                                    (T.pack ("noreply@" ++ uriRegName ourHost))
-                     mail     = (emptyMail mailFrom) {
-                       mailTo      = [Address (Just aname) eml],
-                       mailHeaders = [(BSS.pack "Subject", "[Hackage] " <> subject)],
-                       mailParts   =
-                        [ fromEmailContent $ emailContent <> updatePreferencesText
-                        ]
-                     }
-                     Just ourHost = uriAuthority serverBaseURI
+    sendNotifyEmailAndDelay :: Mail -> IO ()
+    sendNotifyEmailAndDelay email = do
+      -- TODO: if we need any configuration of sendmail stuff, has to go here
+      renderSendMail email
 
-                 renderSendMail mail --TODO: if we need any configuration of
-                                     -- sendmail stuff, has to go here
-                 threadDelay 250000
-      where
-        updatePreferencesText =
-          EmailContentParagraph $
-            "You can adjust your notification preferences at" <> EmailContentSoftBreak
-            <> emailContentUrl
-                serverBaseURI
-                  { uriPath =
-                      concatMap ("/" <>)
-                        [ "user"
-                        , display $ Users.userIdToName users uid
-                        , "notify"
-                        ]
-                  }
+      -- delay sending out emails, because ???
```

Comment:
It's something @gbaz requested, I think it's because we are afraid that we'd be labelled as spammers if we somehow accidentally sent out too many e-mails.